src/app/(auth)/signin/page.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+import { useAuthStore } from '@/stores/useAuthStore'
+import React from 'react'
+import Image from 'next/image'
+import { Logo } from '@assets/icons'
+import Input from '@components/forms/inputGroups/Input'
+import { LoginButton } from '@components/elements/buttons/LoginButton'
+import useStore from '@hooks/useStore'
+import { useRouter } from 'next/navigation'
+
+function Login() {
+  // MARK :: zustand persist 적용 시, Error 발생
+  const router = useRouter()
+  const store = useStore(useAuthStore, (state) => state)
```

Comment:
`Hydration failed because the initial UI does not match what was rendered on the server` 에러 처리를 위한 useStore Hook 생성하였습니다. 사용 방법은 위와 같습니다.

store로 꺼내쓸 때, type에 undefined 추론이 되어서 하단에 `if(!store) return <></>;` 를 추가해뒀습니다.